guide/english/cplusplus/vector/index.md

```diff
@@ -210,6 +227,26 @@ int main()
     return 0;
 }
 ```
+For more complex lambda functions, you can seperate them into an independent function as such.
+```cpp11
+#include <vector>
```

Comment:
I get a few errors when running this code. Could you fix those and update your PR?

Reply:
@scissorsneedfoodtoo This PR has gone stale.  Do you want to fix on behalf or just close it?

Reply:
@RandellDawson, thank you for checking in on this. I'll go ahead and close this since it's been so long since we've had an update.